### DISCUSSION TITLE
Why is blues from light to dark while viridis is from dark to light?

### DISCUSSION THREAD

**Discussion by richardliu-db** [d3/d3#3815] [Q&A] [answered]
Is this intentional? Is there a reason behind this decision? Thanks!
<img width="754" alt="image" src="https://github.com/d3/d3/assets/149440704/8f449b7c-901b-4c48-a96b-786c49388c21">
<img width="760" alt="image" src="https://github.com/d3/d3/assets/149440704/bba19de7-2c8e-4f10-a62a-b6673af1b9e8">

**Comment by mbostock** [comment, marked as answer]
They were designed by different people, for one. 😄 Also relevant is whether the data is being displayed against a dark or a light background. For example, *blue* is good against a light background, but not suitable against a dark one. *viridis* can probably be used in either (though tends to be used for heatmaps that fill an entire region with a quantitative color encoding).
